client/src/app/appCtrl.js

```diff
@@ -15,8 +15,15 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
       threshold : appConfig.MAX_FILE_SIZE,
       show : false,
       paused : false,
-      aborted : false
-    }
+      aborted : false,
+    },
+    highlighting : { // section for highlighting (anomalie) over a threshold
+      fieldName : "sine", //TODO add UI control
+      threshold : 0.8, // UI control
```

Comment:
@jefffohl can you please add the UI controls for this? I thought like:
* drop down list with series names
* text entry for threshold (with checks on being a number)
* maybe? a color selector, but that is not necessary